website_multi_image/static/lib/slick/slick-theme.css

```diff
@@ -0,0 +1,201 @@
+@charset 'UTF-8';
+/* Slider */
+.slick-loading .slick-list
+{
+    background: #fff url('./ajax-loader.gif') center center no-repeat;
+}
+
+/* Icons */
+@font-face
+{
+    font-family: 'slick';
+    font-weight: normal;
+    font-style: normal;
+
+    src: url('./fonts/slick.eot');
+    src: url('./fonts/slick.eot?#iefix') format('embedded-opentype'), url('./fonts/slick.woff') format('woff'), url('./fonts/slick.ttf') format('truetype'), url('./fonts/slick.svg#slick') format('svg');
```

Comment:
I am not sure is we should add new fonts with this module, those should be managed in the theme itself, to use new icons with this module stand alone just add some fontawsome icons should be enought don't you think?